mne/viz/evoked.py

```diff
@@ -477,7 +477,7 @@ def whitened_gfp(x, rank=None):
 
     if not has_sss:
         evokeds_white[0].plot(unit=False, axes=axes_evoked,
-                              hline=[-1.96, 1.96])
+                              hline=[-1.96, 1.96], show=show)
```

Comment:
I'm pretty sure this should be `show=False`, since on line 527 `show` will be done if the user passes `show=True` into the function